src/components/button/button.tsx

```diff
@@ -1,43 +1,28 @@
-import { ReactNode } from 'react';
+import { ComponentProps } from 'react';
```

Comment:
This is nice! I didn't know about this. I've checked the `d.ts` file and it says to prefer `ComponentPropsWithoutRef` or `ComponentPropsWithRef` but I find them too verbose, so I'd even prefer if we just used the `ComponentProps`.

Reply:
Yeah I am of the same opinion. Refs are such a rare thing to use, that I prefer the less verbose `ComponentProps` even if technically, that component can't yet receive a `ref`.